.github/workflows/test_cra.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review, review_requested]
+  issue_comment:
+    types: [created]
+
+jobs:
+  code_review_job:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+      contents: write
+    name: Run code review agent on every pull request, respond to user comments
+
+    steps:
+      - name: Set Options Environment Variable for Issue Comment
+        if: github.event_name == 'issue_comment'
+        run: |
+          echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }}${{ secrets.GIT_DOMAIN && ' --git.domain=' || '' }}${{ secrets.GIT_DOMAIN }}" >> $GITHUB_ENV
+      - name: Code Review Agent - Issue Comment
+        if: github.event_name == 'issue_comment'
+        uses: gitbito/codereviewagent@BITO_QA_TEST
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DjangoProject/pull/5#issuecomment-2168349471>#d58ef4</a><span> - 06/14/2024, 04:18 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The GitHub Action is using the 'gitbito/codereviewagent@BITO_QA_TEST' action. Using a branch that is not intended for production (e.g., a QA or test branch) can introduce instability and unexpected behavior in the workflow.
</div>
<div id="fix">
<b>Fix:</b> Pin the action to a specific version or commit SHA that is intended for production use to ensure stability and prevent unexpected issues due to changes in the QA or test branch.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -23,7 +23,7 @@
       if: github.event_name == 'issue_comment'
       uses: gitbito/codereviewagent@v1.0.0
       with:
         pr: ${{ github.event.issue.pull_request.html_url }}
         command: ${{ github.event.comment.body }}
         options: ${{ env.OPTIONS }}
```
</div>
</details>
</div>
<br>


</div>